kubemarine/resources/scripts/etcdctl.sh

```diff
@@ -67,32 +63,8 @@ if [ -n "${ETCD_POD_CONFIG}" ]; then
   if [ -n "$ETCD_ENDPOINTS" ]; then
     USER_ARGS+=("--endpoints=$ETCD_ENDPOINTS")
   fi
-
-  if [ "$CONT_RUNTIME" == "podman" ]; then
-    # Check if the registry needs authentication:
-    # Match the registry from etcd image with the list of registries that assume an athentication
-    REGISTRIES=$(cat /etc/containerd/config.toml | grep '\.auth\]' | sed 's/.\+configs\."\(.\+\)"\.auth\]/\1/')
-    ETCD_REGISTRY=$(echo ${ETCD_IMAGE} | cut -d "/" -f1)
-    IS_AUTH=$(echo "${REGISTRIES}" | grep ${ETCD_REGISTRY} | wc -l)
-    if [ $IS_AUTH -eq 1 ]; then
-      # Login into registries and pull image if the authentication file exists
-      export REGISTRY_AUTH_FILE=${REGISTRY_AUTH_FILE:-/etc/containers/auth.json}
-      if [ -e ${REGISTRY_AUTH_FILE} ]; then
-	podman login ${ETCD_REGISTRY} > /dev/null 2&>1
-	podman pull ${ETCD_IMAGE} > /dev/null 2&>1
-        podman run --network=host --rm ${ETCD_MOUNTS} -e ETCDCTL_API=3 ${ETCD_IMAGE} \
-		etcdctl --cert=${ETCD_CERT} --key=${ETCD_KEY} --cacert=${ETCD_CA} "${USER_ARGS[@]}"
-      else
-	exit 1
-      fi
-    else
-      podman pull ${ETCD_IMAGE} &> /dev/null
-      podman run --network=host --rm ${ETCD_MOUNTS} -e ETCDCTL_API=3 ${ETCD_IMAGE} \
-	    etcdctl --cert=${ETCD_CERT} --key=${ETCD_KEY} --cacert=${ETCD_CA} "${USER_ARGS[@]}"
-    fi
-  else
-    docker run --rm ${ETCD_MOUNTS} -e ETCDCTL_API=3 ${ETCD_IMAGE} \
-	    etcdctl --cert=${ETCD_CERT} --key=${ETCD_KEY} --cacert=${ETCD_CA} "${USER_ARGS[@]}"
-  fi
+	ctr image pull ${ETCD_IMAGE} > /dev/null 2&>1
+	ctr run --net-host --rm ${ETCD_MOUNTS} --env ETCDCTL_API=3 ${ETCD_IMAGE} etcdctl \
```

Comment:
In case of simultaneous running of 2 or more `etcdctl` commands there could be errors like 
```
ctr: container "etcdctl": already exists 
```